lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -0,0 +1,46 @@
+"""
+Tests for Outline Tab API in the Course Home API
+"""
+
+
+import ddt
+
+from django.urls import reverse
+
+from course_modes.models import CourseMode
+from lms.djangoapps.course_home_api.tests.utils import BaseCourseHomeTests
+from student.models import CourseEnrollment
+
+
+@ddt.ddt
+class OutlineTabTestViews(BaseCourseHomeTests):
+    """
+    Tests for the Outline Tab API
+    """
+    @classmethod
+    def setUpClass(cls):
+        BaseCourseHomeTests.setUpClass()
+        cls.url = reverse('course-home-outline-tab', args=[cls.course.id])
+
+    @ddt.data(CourseMode.AUDIT, CourseMode.VERIFIED)
```

Comment:
Do we have different behavior based on enrollment mode? Might not need to test both of these cases and just choose one enrollment mode.

Reply:
Although might be good to keep it this way so when we add in other components to the course outline that do have a difference, this can take it into account.